src/Dfc.CourseDirectory.WebV2/Features/NewApprenticeshipProvider/ProviderDetailInfoPanel.cshtml

```diff
@@ -1,53 +1,56 @@
 ﻿@model Dfc.CourseDirectory.WebV2.Features.NewApprenticeshipProvider.ProviderDetailInfoPanel.ViewModel
 
-<h2 class="govuk-heading-m govuk-!-margin-0">Contact details</h2>
-<aside class="pttcd-new-apprenticeship-provider__provider-detail-info-panel" role="complementary">
-    @if (Model.AddressParts.Count > 0)
-    {
-        <div class="govuk-!-margin-bottom-4" id="pttcd-new-apprenticeship-provider__provider-detail-info-panel__provider-address">
-            <h3 class="govuk-heading-m m20 govuk-!-margin-0" >Address</h3>
-            <div class="govuk-!-font-size-16">
-                @foreach (var part in Model.AddressParts)
-                {
-                    @part<br />
-                }
+@if (Model.GotContact)
+{
+    <h2 class="govuk-heading-m govuk-!-margin-0">Contact details</h2>
+    <aside class="pttcd-new-apprenticeship-provider__provider-detail-info-panel" role="complementary">
+        @if (Model.AddressParts.Count > 0)
+        {
+            <div class="govuk-!-margin-bottom-4" id="pttcd-new-apprenticeship-provider__provider-detail-info-panel__provider-address">
+                <h3 class="govuk-heading-m m20 govuk-!-margin-0" >Address</h3>
+                <div class="govuk-!-font-size-16">
+                    @foreach (var part in Model.AddressParts)
+                    {
+                        @part<br />
+                    }
+                </div>
             </div>
-        </div>
-    }
-    @if (!string.IsNullOrEmpty(Model.Website))
-    {
-        <div class="govuk-!-margin-bottom-4" id="pttcd-new-apprenticeship-provider__provider-detail-info-panel__provider-website">
-            <h3 class="govuk-heading-m m20 govuk-!-margin-0" >Web</h3>
-            <div class="govuk-!-font-size-16">
-                <a href="@Model.Website" class="govuk-link" rel="noreferrer" target="_blank">@Model.Website</a>
+        }
+        @if (!string.IsNullOrEmpty(Model.Website))
```

Comment:
Is this if still required?

Reply:
Yep - you can have a non-null contact where this field is empty